docs/book/src/reference/ports.md

```diff
@@ -0,0 +1,7 @@
+## Default Ports used by Cluster API / Cluster API Provider
```

Comment:
```suggestion
## Default Ports used by Cluster API / Cluster API Providers
```

Reply:
Let's retitle this as well to be more focused on CAPI, no need to call out providers